lib/payments.js

```diff
@@ -25,6 +25,18 @@ async function getActiveBilling (accountId) {
   return false
 }
 
+async function hasPayedAccount (accountId) {
+  if (!accountId) return false
+  const { payments } = await dbs()
+  try {
+    const doc = await payments.get(String(accountId))
+    if (doc.plan === 'free') return false
+  } catch (error) {
+    if (error.status !== 404) throw error
```

Comment:
🤔 maybe return `false` and log an error? that way this won’t break things, but we can find bugs?